microsoft/testsuites/ltp/ltp.py

```diff
@@ -195,9 +195,7 @@ def _install(self) -> bool:
                 isinstance(self.node.os, Redhat)
                 and self.node.os.information.release >= "8.0"
             ):
-                self.node.os.install_packages(["db4-utils"])
-            else:
-                self.node.os.install_packages(["ntp"])
+                self.node.os.install_packages(["db4-utils", "ntp"])
```

Comment:
As we discussed, make the logic straightforward, or add more comments to explain the expected behavior.

Reply:
I have added the comments to explain this. For the if-else, the current flow should be understandle with comments